corehq/apps/app_manager/templates/app_manager/releases.html

```diff
@@ -215,7 +216,7 @@ <h6 data-bind="if: !built_with.signed()">{% trans "Unsigned Jar" %}</h6>
                             <a class="btn" data-bind="
                                 openModal: 'deploy-build-modal-template',
                                 css: {'btn-primary': !build_broken(), 'btn-danger': build_broken},
-                                click: function() { ga_track_event('App Manager', 'Deploy Button', '{{ app.id }}'); }
+                                click: function() { ga_track_event('App Manager', 'Deploy Button', '{{ app.id }}'); analytics.identify('Clicked Deploy-Button');}
```

Comment:
Bad inline code, but being ad-hoc, alternative is to put all of that into a separate function, but that might be deleted, either because one of GA/Segment would be completely abandoned or this event would be tracked only in one of the them or because all of tracking would be consolidated into one single analytics wrapper.